tests/javascript/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "joomla-cms",
+  "version": "1.0.0",
+  "description": "Joomla! CMS™ [![Analytics](https://ga-beacon.appspot.com/UA-544070-3/joomla-cms/readme)](https://github.com/igrigorik/ga-beacon)\r ====================",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Thanuditha Ruchiranga Wickramasinghe",
+  "license": "GNU",
+  "private": true,
+  "devDependencies": {
+    "jasmine-core": "^2.4.1",
+    "jasmine-jquery": "^2.1.1",
+    "jquery": "^2.2.3",
```

Comment:
Open for discussion: We are using a newer jQuery version then supplied by Joomla itself here. That could lead to other behavior in the libraries we test. Probably we can't work around that, but we should keep that in mind.